projects/github-actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -507,19 +507,11 @@ The e2e test report can be found [here](https://automattic.github.io/jetpack-e2e
 **Follow this PR Review Process:**
 
 1. Ensure all required checks appearing at the bottom of this PR are passing.
-2. Choose a review path based on your changes:
-    - **A. Team Review:** add the "[Status] Needs Team Review" label
-        - For most changes, including minor cross-team impacts.
-        - Example: Updating a team-specific component or a small change to a shared library.
-    - **B. Crew Review:** add the "[Status] Needs Review" label
-        - For significant changes to core functionality.
-        - Example: Major updates to a shared library or complex features.
-    - **C. Both:** Start with Team, then request Crew
-        - For complex changes or when you need extra confidence.
-        - Example: Refactor affecting multiple systems.
-3. Get at least one approval before merging.
-
-Still unsure? Reach out in #jetpack-developers for guidance!`;
+2. Make sure to test your changes on all platforms that it applies to. *You're responsible for the quality of the code you ship*.
+3. Label this PR with the "[Status] Needs Review" label.
+4. When it's reviewed and merged, be ready to deploy the changes to WordPress.com once the build is done.
```

Comment:
Maybe worth being more specific?

```suggestion
4. When it's reviewed and merged, you will be pinged in Slack to deploy the changes to WordPress.com simple once the build is done.
```